packages/app-project/src/screens/ClassifyPage/components/YourProjectStats/YourProjectStats.js

```diff
@@ -0,0 +1,132 @@
+import { Box, ResponsiveContext, Text } from 'grommet'
+import Loader from '@zooniverse/react-components/Loader'
+import SpacedText from '@zooniverse/react-components/SpacedText'
+import { arrayOf, bool, object, number, shape, string } from 'prop-types'
+import { useContext } from 'react'
+
+import ContentBox from '@shared/components/ContentBox'
+import RequireUser from '@shared/components/RequireUser/RequireUser.js'
+import { useTranslation } from 'next-i18next'
+
+function Stat({ label = '', value = 0 }) {
+  return (
+    <Box>
+      <SpacedText textAlign='center'>{label}</SpacedText>
+      <Text
+        color={{ light: 'neutral-1', dark: 'accent-1' }}
+        size='xxlarge'
+        textAlign='center'
+      >
+        {/* Insert commmas where appropriate */}
+        {value.toLocaleString()}
+      </Text>
+    </Box>
+  )
+}
```

Comment:
This Stat presentational component is purposely different than the `@shared/components/Stat` in app-project. The styling here matches lib-user, and the lack of animation is also intentional.